ws-nextjs-app/pages/[service]/send/[id]/FormScreen/index.tsx

```diff
@@ -55,6 +56,7 @@ export default function FormScreen({
     title,
     hasValidationErrors,
     hasAttemptedSubmit,
+    // refocuses on error summary box after every submission attempt
```

Comment:
Appreciate that using attemptedSubmitCount is a bit messy. Though this handles the edge case if the user tries to resubmit the form without editing any of the input fields. Can remove the comment is we think it's better to do so

Reply:
Think this is ok 👍 Probably not something to over-engineer too much given the limited use / edge case.